server/routes/teamEventAPI.js

```diff
@@ -0,0 +1,66 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// 1 - Create a new team event type
+router.post("/api/team-event", auth, (req, res) => {
+  if (req.body.duration) {
+    //
+    db.TeamEventType.create({ ...req.body, teamId: req.teamId })
+      .then((response) => res.send(response))
+      .catch((error) => {
+        console.log(error);
+        res.status(500).send(error);
+      });
+  } else if (!req.body.duration) {
+    res.status(400).send("Duration is required");
+  }
+});
+
+// 2 - Add or remove members from an event type
+// an updated team member is passed to DB to update added and removed members 
+router.post("/api/team/id/:id", (req, res) => {
+  db.Team.update({ _id: req.params.id })
```

Comment:
I think you'll need a second param here to update, it will probably be something like `db.Team.update({ _id: req.params.id }, { $push: { teamMembers: req.body.memberId}}` 
Just made up some variable names, but `$push` because I think teamMembers will be an array and memberId is on the req.body because I find that easier than doing params.
https://mongoosejs.com/docs/api.html#model_Model.update
$push and $set are atomic operators
https://docs.mongodb.com/manual/tutorial/model-data-for-atomic-operations/#pattern